src/main/java/com/mraof/minestuck/network/RemoteObserverPacket.java

```diff
@@ -59,6 +63,7 @@ public void execute(ServerPlayerEntity player)
 					((RemoteObserverTileEntity) te).setActiveType(activeType);
 					if(entityType != null)
 						((RemoteObserverTileEntity) te).setCurrentEntityType(entityType);
+					((RemoteObserverTileEntity) te).setObservingRange(observingRange);
```

Comment:
At this point, I'd suggest using a `RemoteObserverTileEntity observer = (RemoteObserverTileEntity) te;`